### DISCUSSION TITLE
Display under Parameter

### DISCUSSION THREAD

**Discussion by feldsoft** [Louisvdw/dbus-serialbattery#1049] [General]
I just updated serial battery, last version. Setup no. 1.

Now the screen under parameter  is confusing. Before I had one line after the other.

Then I updated  it again to the beta version 1.2.20240227beta. Same screen.

![1](https://github.com/Louisvdw/dbus-serialbattery/assets/143729561/f4415d68-f3b3-40b5-8b44-1079f22a9cbc)

I thing I can not make anything wrong.

JBD BMS 16S

Any idea, was is going on?

Fred

**Comment by mr-manuel** [comment]
Probably you enabled DEBUG in the config file.

**Comment by feldsoft** [comment]
Thank you very much for the solution. Yes I did. I have made an update and after I did that, the second BMS dosen't work. Then I enabled Debug. In the log-file, I found a line: instance or Id of the BMS was found twice. Then I changed the capacity in the config of the BMS from 2800000 to 281000 and everything was fine.

But my real problem is that the current limitation doesn't work. That's why I uninstalled the battery aggregator, then disconnected the second BMS. Enter the serial battery in DVCS and system setup. In the config MAX_BATTERY_CHARGE_CURRENT = 2.0 set to 2 A. My expectation is that it will then only charge with 2 A. But it loads what the panels have to offer. Am I wrong or have I overlooked a setting?
I've been messing around with this for days and can't find the error/setting.

Fred

**Reply by mr-manuel** [reply]
Without more informations it's hard to help you. Best would be, if you open a support issue.
